Dockerfile

```diff
@@ -46,10 +46,66 @@ LABEL name="Application Navigator" \
       description="This image contains the UI for Application Navigator"
 
 USER root
-RUN yum -y remove mariadb-devel
-RUN mkdir -p /opt/ibm/kappnav-ui/app && chown -R 1001:0 /opt/ibm
-COPY --chown=1001:0 licenses/ /licenses/
-WORKDIR /opt/ibm/kappnav-ui/app
+
+ENV NODE_VERSION 8.11.4
+
+ENV YARN_VERSION 1.13.0
+
+# Install node
+RUN microdnf update -y \
+  && microdnf -y install shadow-utils tar xz gzip \
+  && useradd --uid 1001 --gid 0 --shell /bin/bash --create-home node \
+  && ARCH=$(uname -p) \
+   && if [ "$ARCH" != "ppc64le" ] && [ "$ARCH" != "s390x" ]; then \
+     ARCH="x64" ; \
+  fi \
+  # gpg keys listed at https://github.com/nodejs/node#release-keys
+  && set -ex \
+  && for key in \
+    94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
+    FD3A5288F042B6850C66B31F09FE44734EB7990E \
+    71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
+    DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
+    C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+    B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+    77984A986EBC2AA786BC0F66B01FBB92821C587A \
+    8FCCA13FEF1D0C2E91008E09770F7A9A5AE15600 \
+    4ED778F539E3634C779C87C6D7062848A1AB005C \
+    A48C2BEE680E841632CD4E44F07496B3EB3C1762 \
+    B9E2F5981AA6E0CD28160D9FF13993A75599653C \
+  ; do \
+    gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
+    gpg --batch --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
+    gpg --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
+  done \
+  && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-$ARCH.tar.xz" \
+  && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
+  && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
+  && grep " node-v$NODE_VERSION-linux-$ARCH.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
```

Comment:
Verified these are the steps to confirm the node download is signed.
https://github.com/nodejs/node#verifying-binaries